e2e-tests/shared-test-assertions/monetary-field-validation/index.js

```diff
@@ -0,0 +1,90 @@
+import { MINIMUM_CHARACTERS } from '../../constants';
+import { field as fieldSelector } from '../../pages/shared';
+
+/**
+ * monetaryFieldValidation
+ * Assert monetary field validation
+ * @param {String} fieldId: Monetary field ID
+ * @param {Integer} errorIndex: Index of the summary list error
+ * @param {Object} errorMessages: Percentage error messages
+ * @param {Integer} totalExpectedErrors: Total expected errors in the form
+ * @param {Integer} totalExpectedOtherErrorsWithValidMonetaryValue: Total expected errors in the form when the monetary field is valid.
+ * @param {Boolean} minimum: Minimum allowed value.
+ * @returns {Function} Mocha describe block with assertions.
```

Comment:
we don't have this for every command (returns).  Do we need it here?

Reply:
Yes, because it's a shared test assertion function - it returns mocha describe blocks